packages/core-transaction-pool-mem/package.json

```diff
@@ -21,6 +21,7 @@
     "depcheck": "depcheck ./"
   },
   "dependencies": {
+    "@arkecosystem/core-api": "^0.1.1",
```

Comment:
Why is this required by the transaction pool? The transaction pool shouldn't have a dependency on anything other then the container and pool interface 👀

Reply:
To convert "number of seconds since genesis block" to "number of seconds since 1970". Now changed in https://github.com/ArkEcosystem/core/pull/1066/commits/777ac2222482164f9c5a9f550e385e896476e275